.github/workflows/ci.yaml

```diff
@@ -8,27 +8,13 @@ on:
       - "*"
   workflow_dispatch: # allows you to trigger manually
 
-jobs:
-  skip-duplicate-jobs:
-    runs-on: ubuntu-latest
-    if: |
-      github.repository == 'pydata/xarray'
-      && (github.event_name == 'push' || github.event_name == 'pull_request')
```

Comment:
@aaronspring, we may want to keep this condition so as to avoid running the CI on forks

Reply:
Shouldn’t we keep this? @max-sixty @andersy005

Reply:
we should! @aaronspring, can you open a new PR for that?

Reply:
Sorry, I merged too early. @aaronspring if you can open a PR then great, otherwise I can as penance

Reply:
I put the condition in `detect-ci-trigger` in #6223